src/components/LoginModal.tsx

```diff
@@ -0,0 +1,214 @@
+import {
```

Comment:
Actual:
![image](https://github.com/user-attachments/assets/79931ab2-0c40-4f32-a8c5-2aa7e66b05f2)

Suggestions:
- Decrease the width (same as Login would be good)
- Make the eye icon white

Reply:
@SamuelPelletierEvraire still too large.